tests/legacy/ApiTest.php

```diff
@@ -2557,8 +2560,9 @@ public function testApiHelpTest()
 	 */
 	public function testApiHelpTestWithXml()
 	{
-		$result = api_help_test('xml');
-		self::assertXml($result, 'ok');
+		// @todo How to test the new API?
```

Comment:
You can call `\Friendica\Module\Api\Friendica\Help\Test::rawcontent(['extension' => 'xml'])` instead.

We don't need `BaseApi::$format` anymore since the format is now available in the router parameters, it's replaced with `$parameters['extension'] ?? 'json'` in the `rawContent` methods.

Reply:
The function doesn't have a return value. How can we do the compare?

Reply:
Ah, you're right, I thought it was like the `BaseModule::content()` method. I'm stumped too, then.

Reply:
Ah, you're right, I thought it was like the `BaseModule::content()` method. I'm stumped too, then.